food-truck-api/src/main/resources/application-dev.properties

```diff
@@ -1,3 +1,4 @@
 spring.datasource.url=jdbc:mysql://localhost:3308/foodtruckdb
 spring.datasource.username=root
-spring.datasource.password=password
\ No newline at end of file
+spring.datasource.password=password
```

Comment:
I'm kind of confused about this. What's the spring datasource, and why is its password on Github? I'm guessing this is a dev password that slipped through.

Reply:
This is the dev database password. It's also in the `docker-compose.yml` file. Not really an issue to have this committed to the repo.